app/services/user_service.rb

```diff
@@ -0,0 +1,58 @@
+module UserService
```

Comment:
Maybe consider using `class << self` to get rid of the `self.`s. It also makes it easier to make the private methods actually private:

```ruby
module UserService
  class << self
    def create; end

    private

    def _user_from_params; end
  end
end
```

Reply:
This is done